wurst/objects/items/Items.wurst

```diff
@@ -122,4 +77,30 @@ public class ItemChannelCast extends ChannelAbilityPreset
         this.setManaCost(1, 0)
         this.setCooldown(1, cooldown)
         this.setCastRange(1, castRange)
+        this.setTargetsAllowed(1, targetsAllowed)
         this.setEditorSuffix("(Wurst)")
+
+/**
+    Use this to create an item spell who doesn't play an animation
+    Contrary to the Channel ability, WindWalk do not stop the unit current order when casted.
+**/
+public class ItemWindWalkCast extends AbilityDefinitionWindWalk
```

Comment:
Why are you using the `extends` pattern here?

Reply:
No real reason except I wrote it months ago and copy pasted it, edited.